src/services/filterManager.ts

```diff
@@ -128,6 +129,94 @@ export const saveToLocalStorage = (filter, index, collection) => {
   )
 }
 
+export const removeHistoryRawFromLocalStorage = (id, index, collection) => {
```

Comment:
If you use the word `History` you state that the function removes the whole history from localStorage. If it just removes one filter, the name should be
```suggestion
export const removeFilterFromLocalStorage = (id, index, collection) => {
```